mainapp/models.py

```diff
@@ -36,23 +36,27 @@ class Responce(models.Model):
     message = models.CharField('message', max_length=100, blank=False)
     created_at = models.DateTimeField('created_at', default=datetime.utcnow)
     deleted_at = models.DateTimeField('deleted_at', null=True)
-    product = models.OneToOneField(Product, on_delete=models.SET_NULL, null=True)
-    autor = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
+    product = models.ForeignKey('Product', on_delete=models.SET_NULL, null=True)
+    autor = models.ForeignKey('Profile', on_delete=models.SET_NULL, null=True)
 
     def __str__(self):
         return self.message
 
 
 class Profile(models.Model):
+    deleted_time = datetime.now() + timedelta(days=10)
```

Comment:
It doesn't works in this way. `deleted_time` will be calculated only once after app starting.
You can create separated function for default value

```python3
from django.db import models

def autodelete_date(delta=timedelta(days=10)):
    return F('updated_at') - delta


class Profile(models.Model):
    updated_at = models.DateTimeField('updated_at', auto_now=True, auto_now_add=True)
    deleted_at = models.DateTimeField('deleted_at', default=autodelete_date)
```

By the way, it's acceptable for deactivating `Products` but auto-deleting `Profiles` is bad idea.